app/controllers/admin/courts_controller.rb

```diff
@@ -129,5 +131,13 @@ def area_of_law_by_id_or_name
       return area_of_law if area_of_law.present?
       AreaOfLaw.where(name: AreaOfLaw::Name::CHILDREN).first
     end
+
+    def process_update_exception(e)
+      if e.message.include?('postcode_court')
```

Comment:
is this something we could move into the [Concerns::Court::PostcodeCourts](https://github.com/ministryofjustice/courtfinder/blob/old_prod_working//app/models/concerns/court/postcode_courts.rb) module?